packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-editor/properties-section/property-table/cells/value-cell/value-cell-editor.tsx

```diff
@@ -0,0 +1,25 @@
+import { ProvideEditorComponent } from "@glideapps/glide-data-grid";
+import { TextField } from "@hashintel/hash-design-system";
+import { ValueCell } from "../value-cell";
+
+/**
+ * @todo this should be used only for strings
+ * make sure to handle editors for other types as well
+ * */
+export const ValueCellEditor: ProvideEditorComponent<ValueCell> = ({
+  value,
+  onChange,
+}) => {
+  return (
+    <TextField
+      sx={{ my: "1px" }}
+      autoFocus
+      value={value.data.property.value}
+      onChange={(event) => {
+        const newValue = { ...value };
+        newValue.data.property.value = event.target.value;
```

Comment:
This mutates `value`, which you appear to have attempted to avoid

Reply:
Good catch, used `lodash.cloneDeep` instead